Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityInputSystemProfileInspector.cs

```diff
@@ -260,7 +248,8 @@ private void RenderList(SerializedProperty list)
                             }
                         }
 
-                        if (providerFoldouts[i] || RenderAsSubProfile)
+                        // TODO: TROY
```

Comment:
> // TODO: TROY [](start = 23, length = 14)

Is it the case that the RenderAsSubProfile section can just be deleted now?

Reply:
This comment is on an outdated version of the file. It has already been removed